static/js/stream_list.js

```diff
@@ -8,6 +8,8 @@ var private_messages_open = false;
 var last_private_message_count = 0;
 var last_mention_count = 0;
 var previous_sort_order;
+var previous_unstarred_order;
+var starred_streams;
```

Comment:
This doesn't need to be a global variable. Make it local to `exports.build_stream_list`